torchaudio/transforms/_transforms.py

```diff
@@ -1557,6 +1559,18 @@ def __init__(
         self.hop_length = hop_length if hop_length is not None else self.win_length // 4
         window = window_fn(self.win_length) if wkwargs is None else window_fn(self.win_length, **wkwargs)
         self.register_buffer("window", window)
+        self.rate = 2.0 ** (-float(n_steps) / bins_per_octave)
+        self.orig_freq = int(sample_rate / self.rate)
+        self.new_freq = sample_rate
+        self.gcd = math.gcd(int(self.orig_freq), int(self.new_freq))
+
+        if self.orig_freq != self.new_freq:
```

Comment:
this is outside the scope of this PR, but for someone to follow up on later -- could be good to integrate the `orig_freq == new_freq` case into the resampling helper methods to avoid needing this if statement in various code places